openedx2zim/xblocks_extractor/Discussion.py

```diff
@@ -1,39 +1,34 @@
-import os
 import re
-import bs4 as BeautifulSoup
-from slugify import slugify
-from ..utils import make_dir, jinja
 
+from bs4 import BeautifulSoup
 
-class Discussion:
-    is_video = False
+from .base_xblock import BaseXblock
+from ..utils import jinja
 
-    def __init__(self, json, path, rooturl, id, descendants, mooc):
-        self.mooc = mooc
-        self.json = json
-        self.path = path
-        self.rooturl = rooturl
-        self.id = id
-        self.output_path = self.mooc.output_path
-        path = os.path.join(self.output_path, self.path, slugify(json["display_name"]))
-        make_dir(path)
+
+class Discussion(BaseXblock):
+    def __init__(self, xblock_json, relative_path, root_url, id, descendants, scraper):
+        super().__init__(xblock_json, relative_path, root_url, id, descendants, scraper)
+
+        # extra vars
         self.data = []
         self.category_title = ""
+        self.is_video = False
 
     def download(self, c):
-        if self.mooc.forum_thread != None:
```

Comment:
Have you checked this value? `None` evaluates to `False` but most `False` value are different from `None`